hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockCacheFactory.java

```diff
@@ -17,9 +17,6 @@
  */
 package org.apache.hadoop.hbase.io.hfile;
 
-import static org.apache.hadoop.hbase.HConstants.BUCKET_CACHE_IOENGINE_KEY;
-import static org.apache.hadoop.hbase.HConstants.BUCKET_CACHE_SIZE_KEY;
```

Comment:
Why this two configs got deleted?

Reply:
I accidentally included import static org.apache.hadoop.hbase.HConstants.* and removed the above two ones and didnt notice that. But now it is clean. Thanks.

Reply:
Please find the latest commit with the changes.